### ISSUE TITLE
Implement ScoreForSession for FiddlerInspector

### ISSUE THREAD

**Issue by ericlaw1979** [SteveSyfuhs/Kerberos.NET#174]
By implementing this method, you signal to Fiddler that your Inspector is especially relevant for a given session. For the Kerberos Inspector, a strong clue is that the response is a HTTP/401 and there was a `WWW-Authenticate` header.

Something like:

```
        public override int ScoreForSession(Session oS)
        {
                if ((401 == oS.responseCode && oS.ResponseHeaders["WWW-Authenticate"].OICStartsWith("N")) ||
                    (407 == oS.responseCode && oS.ResponseHeaders["Proxy-Authenticate"].OICStartsWith("N"))) {
                      return 100;
                    }
                //  TODO: Maybe return > 0 if there's a Authorization header on the request even 
                //              if the response status isn't 401/407?
               return 0;
         }
````

**Comment by SteveSyfuhs** [comment]
Interesting. Should be an easy change. There's some additional checks for KDC Proxy that would need to happen, but those are cheap.

What does being especially relevant to a particular session influence in practice?

**Comment by ericlaw1979** [comment]
Hehe, I forgot a sentence. 

When you double-click a Session, Fiddler queries all of the inspectors to find out which Inspector thinks itself most relevant. Double-click a JSON response and the JSON inspector activates. Double-click an image response and the ImageView activates. Etc.

**Comment by SteveSyfuhs** [comment]
🤯 TIL. Super useful.

**Comment by SteveSyfuhs** [comment]
PR is up https://github.com/dotnet/Kerberos.NET/pull/175

**Comment by SteveSyfuhs** [comment]
Merged in #175
